src/js/components/visualization/TableMenu.jsx

```diff
@@ -0,0 +1,26 @@
+import Download from 'material-ui/svg-icons/file/file-download'
+import MenuItem from 'material-ui/MenuItem/MenuItem'
+import React, {Component, PropTypes} from 'react'
+
+export class TableMenu extends Component {
+  static propTypes = {
+    visualization: PropTypes.object.isRequired
+  }
+
+  render () {
+    const {visualization} = this.props
+  
+    return (
+      <span>
+        <MenuItem
+          key='exportExcel'
+          leftIcon={<Download />}
+          primaryText='Export'
+          onTouchTap={function (event) {
```

Comment:
don't define callbacks inline. when you do, they get created on every render call